### ISSUE TITLE
brace-style options for pretty-diff

### ISSUE THREAD

**Issue by amorino** [Glavin001/atom-beautify#1076]
# Description

Customize the brace-style of the beautification, currently using pretty diff, it collapses.
# Input

``` js
import { render } from "react-dom";
```
# Output

``` js
import {render} from "react-dom";
```
# Expected Results

``` js
import { render } from "react-dom";
```
# Steps to Reproduce
1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor` on a .js using ES6
3. This beautified code does not look right!
# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/anonymous/4780ecc468ba3be2c1d27b48f1307a15
# Checklist
- [ ] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [ ] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
  so I know this is not a duplicate issue
- [x] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by prettydiff** [comment]
Pretty Diff has an option for this: [bracepadding](http://prettydiff.com/documentation.xhtml#bracepadding).  I do not see a similar option in the Atom Beautify settings.

**Comment by Glavin001** [comment]
We should add Pretty Diff support for `brace-style` option (see https://github.com/Glavin001/atom-beautify/blob/7eab25e46ffa2b44102e8c1442f2d04ada82b56a/src/languages/javascript.coffee#L72 ) and convert it to `bracepadding` option internally.

**Comment by carlesnunez** [comment]
Any news here?

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1193 added `bracepadding` support via Atom-Beautify option named `space_in_paren`.

It closed https://github.com/Glavin001/atom-beautify/issues/1157 and should have also closed this.

**Comment by Glavin001** [comment]
Published to v0.29.12
